docs/web-code-style.md

```diff
@@ -0,0 +1,20 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+   -
+   - SPDX-License-Identifier: CC0-1.0
+   -->
+
+As foundation for our TypeScript/SCSS styleguide
+[this styleguide](https://basarat.gitbook.io/typescript/styleguide) is used,
+except for the followings:
+
+## Filename
```

Comment:
What about forders?

Reply:
what exactly? I guess `MyComponent/{MyComponent.tsx, MyComponent.scss}` structure will be used, something like htis